addons/website/static/src/js/website.seo.js

```diff
@@ -71,31 +77,39 @@ var SuggestionList = Widget.extend({
     },
     refresh: function () {
         var self = this;
-        self.$el.append("Loading...");
-        function addSuggestions (list) {
-            self.$el.empty();
-            // TODO Improve algorithm + Ajust based on custom user keywords
-            var regex = new RegExp(self.root, "gi");
-            var cleanList = _.map(list, function (word) {
-                return word.replace(regex, "").trim();
-            });
-            // TODO Order properly ?
-            _.each(_.uniq(cleanList), function (keyword) {
-                if (keyword) {
-                    var suggestion = new Suggestion(self, {
-                        root: self.root,
-                        keyword: keyword,
-                        page: self.htmlPage,
-                    });
-                    suggestion.on('selected', self, function (word) {
-                        self.trigger('selected', word);
-                    });
-                    suggestion.appendTo(self.$el);
-                }
-            });
-        }
-        $.getJSON("/website/seo_suggest/" + encodeURIComponent(this.root + " "), addSuggestions);
+        self.$el.append(_t("Loading..."));
+        var language = self.language || website.get_context().lang.toLowerCase();
+        ajax.jsonRpc('/website/seo_suggest', 'call', {
+            'keywords': self.root,
+            'lang': language,
+        }).then(function(list){
+            self.addSuggestions(JSON.parse(list));
+        });
     },
+    addSuggestions: function(list) {
+        var self = this;
+        self.$el.empty();
+        // TODO Improve algorithm + Ajust based on custom user keywords
+        var regex = new RegExp(self.root, "gi");
+        var cleanList = _.map(list, function (word) {
+            return word.replace(regex, "").trim();
+        });
+        // TODO Order properly ?
+        _.each(_.uniq(cleanList), function (keyword) {
```

Comment:
cleanList is not clean, keyword list contain with root keyword, so remove this root keyword from every suggested keyword.
and replace a root keyword to null string so need to check condition for keyword otherwise show 1 keyword tag without string.